src/components/profile/profile-qualification/certificate-form/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { ProfileQualityFormValues } from 'src/components/profile/profile-qualification/types';
+
+export const DATE_FORMAT = 'dd/MM/yyyy';
+
+export const URL_PATTERN = /^https:\/\/.+$/;
```

Comment:
hmmm I already seen this code. you don't make your branch from develop branch/ each PR should be independent in other case it does to make sense in branching

Reply:
fixed